src/app/pages/ArticlePage/ElectionBanner/index.test.tsx

```diff
@@ -25,6 +25,54 @@ describe('ElectionBanner', () => {
   describe.each(['canonical', 'amp'])('%s', platform => {
     const isAmp = platform === 'amp';
 
+    it.each(['live', 'test'])(
+      'should use the correct URL for the iframe when SIMORGH_APP_ENV is "%s"',
+      appEnv => {
+        if (appEnv === 'live') {
+          process.env.SIMORGH_INCLUDES_BASE_URL =
```

Comment:
Do we need to delete / reset the environment variables that we override in a beforeEach or afterEach to prevent them from potentially affecting other tests?